lang-python/python-pdf2docx/autobuild/defines

```diff
@@ -0,0 +1,8 @@
+PKGNAME=python-pdf2docx
+PKGSEC=python
+PKGDEP="python-3 python-pymupdf python-docx python-fonttools python-fire python-termcolor lxml opencv"
```

Comment:
I don't think this really needs lxml directly.  It seems you are just covering up the real issue that python-docx misses the lxml dependency here.

Don't do that.